app/(main)/settings/team/agent-time-charge.tsx

```diff
@@ -0,0 +1,23 @@
+import { calculateAgentTimeUsage } from "@/services/agents/activities/agent-time-usage";
+import { fetchCurrentTeam } from "@/services/teams";
+import {
+	AgentTimeUsageForFreePlan,
+	AgentTimeUsageForProPlan,
+} from "./components/agent-time-usage";
+
+export async function AgentTimeCharge() {
+	const currentTeam = await fetchCurrentTeam();
+	const isProPlan =
```

Comment:
I thought it would be better to use the following!

https://github.com/giselles-ai/giselle/blob/53ed3f50aa76f4a1accb5d897687e34cb037dcfe/services/teams/utils.ts#L3-L5

Reply:
⚒️  [adcf400](https://github.com/giselles-ai/giselle/pull/200/commits/adcf400da63095c9cb97c8b283a36ef1abd6c15a)